tool/tsh/tsh.go

```diff
@@ -602,6 +605,8 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 
 	ver := app.Command("version", "Print the version of your tsh binary")
 	ver.Flag("format", defaults.FormatFlagDescription(defaults.DefaultFormats...)).Short('f').Default(teleport.Text).EnumVar(&cf.Format, defaults.DefaultFormats...)
+	ver.Flag("client-only", "Only display tsh binary version and do not retrieve proxy version.").Default("false").
```

Comment:
Can probably omit the default since booleans are false by default anyway.

Reply:
removed